### PR TITLE
feat: clarify who can run a bridge

### PR DESCRIPTION
**Pull request by caphesuagey**
### Reason

Closes: N/A

### What's being changed

* Specified who can run a bridge node
* Replaced "bridge operator node" with "bridge node" for correctness
* Expanded the prerequisites and steps in the "Run a bridge node" guide

### Checklist

- [x] I've reviewed my changes in a preview environment (click the link in the "Preview" column to view your latest changes).
- [x] For content changes, I've completed the [self-review checklist](https://github.com/axieinfinity/ronin-documentation/blob/main/docs/CONTRIBUTING.md#self-review-checklist).

### CODE REVIEW

File: docs/bridge-operators/setup/run-bridge.md
Comment (RodinGolodin) [inline docs/bridge-operators/setup/run-bridge.md:1]: @minh-bq Please review this file.

Comment (RodinGolodin) [reply docs/bridge-operators/setup/run-bridge.md:1]: @dwi @seuchenhund

File: docs/bridge-operators/setup/run-bridge.md [hunk 1]
```diff
@@ -111,42 +104,36 @@ The size of your Ronin node will also grow over time.
          - db
    ```
 
-   This compose file defines three services:
+   This compose file defines two services:
 
-     * `bridge`, which pulls a bridge image.
-     * `db`, which builds a Postgres database for the bridge.
+     * `bridge` that build the bridge service.
```
Comment (minh-bq) [inline docs/bridge-operators/setup/run-bridge.md:109]: *builds

File: docs/bridge-operators/setup/run-bridge.md [hunk 2]
```diff
@@ -1,36 +1,47 @@
...
 
 ### Private keys
 
-Generate two private keys as described in [Generate keys](./../../validators/setup/generate-keys.md):
+Generate private keys:
 
-* One key for the bridge voter (if you're a governor)
-* One key for the bridge operator
+* Generate a key for the bridge operator.
+* If you're a Governing Validator, generate one more key for the bridge voter.
```
Comment (minh-bq) [inline docs/bridge-operators/setup/run-bridge.md:38]: The bridge voter key is not needed anymore

File: docs/bridge-operators/setup/run-bridge.md [hunk 3]
```diff
@@ -111,42 +104,36 @@ The size of your Ronin node will also grow over time.
...
+   # The latest version of the bridge's image as listed in https://docs.roninchain.com/bridge-operators/setup/upgrade-bridge
+   BRIDGE_IMAGE=<your_bridge_image_version>
  
    # Your bridge operator private key without the 0x prefix
-   LISTENERS__RONIN__SECRET__BRIDGEOPERATOR__PLAINPRIVATEKEY=LISTENERS__RONIN__SECRET__BRIDGEOPERATOR__PLAINPRIVATEKEY
+   LISTENERS__RONIN__SECRET__BRIDGEOPERATOR__PLAINPRIVATEKEY=<your_bridge_operator_private_key>
  
    # If you're a governor, uncomment this line and replace with your bridge voter key, without the 0x prefix
-   # BRIDGE_VOTER_PRIVATE_KEY=BRIDGE_VOTER_PRIVATE_KEY
+   # BRIDGE_VOTER_PRIVATE_KEY=<your_bridge_voter_private_key>
```
Comment (minh-bq) [inline docs/bridge-operators/setup/run-bridge.md:125]: Please remove this variable, bridge voter key is not needed anymore

File: docs/bridge-operators/setup/run-bridge.md [hunk 4]
```diff
@@ -182,9 +170,11 @@ Whenever you wish to verify that your node is working, run the following comman
 docker-compose logs bridge | head -n 20
 ```
 
-Verify that the operator account address in the response matches your registered
+Make sure that the "Operator account" address in the response matches your registered
 bridge operator address. If you're a Governing Validator, also check that the
-voter account address matches the registered bridge voter address.
+"Voter account" address matches your registered bridge voter address.
```
Comment (minh-bq) [inline docs/bridge-operators/setup/run-bridge.md:175]: Remove bridge voter part

File: docs/bridge-operators/setup/run-bridge.md
Comment (RodinGolodin) [inline docs/bridge-operators/setup/run-bridge.md:1]: @minh-bq @huyngopt1994

**Review by minh-bq — APPROVED** [review]
The bridge part looks good to me

**Pull request merged**